e2e/v3/jobs3/jobs3.go

```diff
@@ -579,8 +585,26 @@ func Verbose(on bool) Option {
 // Set an HCL variable.
 func Var(key, value string) Option {
 	return func(sub *Submission) {
-		sub.vars.Insert(fmt.Sprintf("%s=%s", key, value))
+		sub.vars[key] = value
+	}
+}
+
+type Vars map[string]string
+
+func (v Vars) Slice() []string {
+	s := make([]string, 0, len(v))
+	for k, v := range v {
+		s = append(s, fmt.Sprintf("%s=%s", k, v))
+	}
+	return s
+}
+
+func (v Vars) String() string {
+	s := ""
+	for k, v := range v {
+		s = s + fmt.Sprintf("%s=%q\n", k, v)
 	}
+	return s
```

Comment:
I had to double-check because I had a reason but forgot since I wrote this last week :yum:  -- it's the `%q` on 605, which quotes the value, e.g.

```
var1="val1"
var2="val2"
```

vs `%s` from Slice()

```
var1=val1
var2=val2
```

the API call to actually register the job wants the latter, but the Submission metadata, which shows up in the UI, wants the former.

Reply:
:+1: